tests/docker/BUILD

```diff
@@ -13,10 +13,25 @@
 # limitations under the License.
 package(default_visibility = ["//visibility:public"])
 
+load("//container:container.bzl", "container_image")
 load("//contrib:test.bzl", "container_test")
 
 container_test(
     name = "structure_test",
     configs = ["//tests/docker/configs:test.yaml"],
     image = "//testdata:link_with_files_base",
 )
+
+container_image(
```

Comment:
It would be nice to have a test that used an actual deb :)

Reply:
will do